coreneuron/nrnconf.h

```diff
@@ -39,9 +39,7 @@ namespace coreneuron {
 
 #define NRNBBCORE 1
 
-typedef int Datum;
-typedef int (*Pfri)();
```

Comment:
In `coreneuron/mechanism/register_mech.cpp` I saw different prototype ?

Reply:
Dito